src/main/java/es/urjc/cloudapps/library/data/InMemoryBookRepository.java

```diff
@@ -1,16 +1,29 @@
 package es.urjc.cloudapps.library.data;
 
 import es.urjc.cloudapps.library.domain.Book;
+import org.springframework.stereotype.Repository;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
 
+@Repository
 public class InMemoryBookRepository {
 
     private Map<String, Book> books;
 
     public InMemoryBookRepository() {
         this.books = new HashMap<>();
+        String uuid1 = UUID.randomUUID().toString();
+        String uuid2 = UUID.randomUUID().toString();
+        this.books.put(uuid1, new Book(uuid1, "Libro 1", "resumen", "cristofer", "shurs"));
+        this.books.put(uuid2, new Book(uuid2, "Libro 2", "resumen", "cristofer", "shurs"));
+    }
+
+    public List<BookDto> findAll() {
```

Comment:
Aquí devolvemos los 'Book' de dominio para que el servicio pueda ejecutar sobre ellos lógica de negocio, esta transformación la debe hacer el servicio de aplicación (BookService)
```suggestion
    public List<Book> findAll() {
```